metadata-models/src/main/pegasus/com/linkedin/identity/CorpUserSettings.pdl

```diff
@@ -0,0 +1,15 @@
+namespace com.linkedin.identity
+
+/**
+ * Settings that a user can customize through the datahub ui
+ */
+@Aspect = {
+  "name": "corpUserSettings"
+}
+record CorpUserSettings {
+  /**
+   * Flag whether the user should see a homepage with only datasets, charts & dashboards. Intended for users
+   * who have less operational use cases for the datahub tool.
+   */
+  showSimplifiedHomepage: optional boolean
```

Comment:
Can we actually put this under an appearances header? This aspect is _undoubtedly_ going to grow and I don't know if it makes sense to have this at top level. Unless all settings + preferences will remain top level. In GlobalSettings we break out into categories. What do you think?